code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -58,83 +56,85 @@ put up a rune with bluespace effects, lots of those runes are fluff or act as a
 			qdel(space_crystal)
 
 
-
-
 /obj/effect/warped_rune/acid_act()
 	visible_message("<span class='warning'>[src] has been dissolved by the acid</span>")
 	playsound(src, 'sound/items/welder.ogg', 150, TRUE)
 	qdel(src)
 
-///nearly all runes use their turf in some way so we set T to their turf automatically, the rune also start on cooldown if it uses one.
+
+///nearly all runes use their turf in some way so we set rune_turf to their turf automatically, the rune also start on cooldown if it uses one.
 /obj/effect/warped_rune/Initialize()
-	.=..()
-	T = get_turf(src)
-	RegisterSignal(T, COMSIG_COMPONENT_CLEAN_ACT, .proc/clean_rune)
+	. = ..()
+	rune_turf = get_turf(src)
+	RegisterSignal(rune_turf, COMSIG_COMPONENT_CLEAN_ACT, .proc/clean_rune)
 	cooldown = world.time + max_cooldown
 
+
 /obj/effect/warped_rune/proc/clean_rune()
 	qdel(src)
 
- ///using the extract on the floor will "draw" the rune.
-/obj/item/slimecross/warping/afterattack(turf/target, mob/user, proximity)
+
+///using the extract on the floor will "draw" the rune.
+/obj/item/slimecross/warping/afterattack(target, user, proximity)
```

Comment:
Don't remove the var types if they are guaranteed to be of those types when called.